StructuralPreprocessing/StructuralPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -49,10 +49,14 @@ get_options()
 	local arguments=($@)
 
 	# initialize global output variables
-	unset g_project
-	unset g_subject
-	unset g_classifier
-	unset g_running
+    unset g_user
+    unset g_password
+    unset g_server
+    unset g_project
+    unset g_subject
+    unset g_session
+    unset g_resource
+    unset g_reason
```

Comment:
Same comment as the FunctionalPreprocessing version of this file. Let's build the `g_reason` value based on the `--submitted`, `--queued`, ... `--done` parameter.

Since this file is so very similar to the `FunctionalPreprocessing.XNAT_MARK_RUN_STATUS` script, maybe we can think of a way to just have a single script that does this work with an additional parameter passed in that specifies the name of the processing that is being marked as running or not running. Like maybe a `--processing-name=<value>` required parameter.  We could call this new script simply `xnat_mark_run_status.sh` and put it in a new subdirectory in the repository called `Utils`? In addition to all the parameters that the `StructuralPreprocessing.XNAT_MARK_RUN_STATUS` script has, it would have the new `--processing-name=` required parameter. To serve the needs of the `FunctionalPreprocessing.XNAT_MARK_RUN_STATUS` script,. the `xnat_mark_run_status.sh` script would have to have an *optional* `--scan=` parameter. If the `--scan=` parameter is not specified, then its value would simply not be included in the name of the marker file. If the `--scan=` parameter is specified, then its value *would* be included in the name of the marker file.

Then `StructuralPreprocessing.XNAT_MARK_RUNNING_STATUS` would still exist and be reduced pretty much to just a call to `xnat_mark_run_status.sh`, and FunctionalPreprocessing.XNAT_MARK_RUNNING_STATUS would still exist and be reduced pretty much to just a call to `xnat_mark_run_status.sh` with different parameter values.  This would make it easy to produce other XNAT_MARK_RUNNING_STATUS scripts (like for DiffusionPreprocessing) as just calls to `xnat_mark_run_status.sh`.

That seem like a good idea to you?